index.html

```diff
@@ -0,0 +1,214 @@
+          <section class="project">
+      <button class="project">See Project</button>
+   </section> 
+  </div>
+  </div>
+       </div>
+      </section>
+        </section>
+<section class="about-me">
+        <div class="about-myself">
+          <div class="bg-images">
+          <img class="img1" src="./images/frame5.png" >
+          <img class="img2" src="./images/frame4.png">
+          </div>
+          <div class="content">
+          <h1 class="about-title">About Me</h1>
+          <p class="support-text">
+             I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+don’t hestiate to contact me.
+          </p>
+           <button class="project">Get my Resume</button>
+            </div>
+          </div >
+         <section class= "grids">
+         <div class="grid">
+            <img class="grid-image" src="./images/language.png" alt="">
+            <div class="txt1">
+            <h3 class="lang-title">Languages</h3>
+             <section class = "tags1">
+            <li class="tag1">Javascript</li>
+            <li class="tag1">Ruby</li>
+            <li class="tag1">Html</li>
+            <li class="tag1">CSS</li>
+          </section>
+          </div>
+         </div>
+          <div class="grid">
+          <img  class="grid-image" src="./images/frameworks.png" alt="">
+          <div class="txt1">
+          <h3>Frameworks</h3>
+           <div class = "tags1">
+            <li class="tag1">Bootstrap</li>
+            <li class="tag1">Ruby on Rails</li>
+            <li class="tag1">Rspec</li>
+           </div>
+             <div class = "tags2">
+            <li class="tag1">Capybara</li>
+            <li class="tag1">Selenium</li>
+          </div>
+        </div>
+          </div>
+       <div class="grid">
+        <img class="grid-image"src="./images/skills.png" alt="">
+        <div class="txt1">
+        <h3>Skills</h3>
+        <div class = "tags1">
+           <li class="tag1">Codekit</li>
+            <li class="tag1">Github</li>
+            <li class="tag1">Codepen</li>
+            <li class="tag1">Gitlab</li>
+  </div>
+            <div class="tags2">
+            <li class="tag1">Terminal</li>
+            </div>
+          </div>
+     </div> 
+         </section>
+    <div class="contact">
+      <div  class="contact-text">
+      <h4>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h4>
+     </div>
+     <div class="contact-right">
+     <div class="btns1">
+      <input class="smallbtn"  type="text" value="Full Name" size="33">
+      </div>
+    <input class="smallbtn3" type="text" value="Email adress" size="33">
+    <textarea class="txt-area">Type something</textarea>
+    <li class="get-in-touch">Get In Touch</li>
+    </div>
+    </section>
+    <section class="social-media">
+          <a href="https://twitter.com/kalungibobsonj">
```

Comment:
- [ ] Kindly center your `contact` form inputs. They are aligned to the left. Also, adjust the `get in touch` button padding to match the Figma design.

![image](https://user-images.githubusercontent.com/98453979/200135898-2d965099-0e07-4c84-aac7-4479c34a7f1f.png)

### Checkout Figma 

![image](https://user-images.githubusercontent.com/98453979/200136038-d5cb2615-0cb6-4671-aa0f-4511671155dd.png)

- [optional ] I also noticed that your portfolio is missing the `headline` and `tool bar` sections. Kindly add them  as they are essential parts of your portfolio